Rhino.Templates/content/CSGrasshopper/.vscode/launch.json

```
@@ -0,0 +1,87 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Launch Rhino 8",
+      "type": "coreclr",
+      "request": "launch",
+      "preLaunchTask": "build",
+      "program": "",
+      "osx": {
+        "program": "/Applications/Rhino 8.app/Contents/MacOS/Rhinoceros"
+      },
+      "windows": {
+        "program": "C:\\Program Files\\Rhino 8\\System\\Rhino.exe",
+        "targetArchitecture": "x86_64",
+      },
+      "args": [],
```

I would love it if Grasshopper opens when I debug a GH Plugin
```suggestion
      "osx": {
            "args": ["-runscript=Grasshopper"],
      },
      "windows": {
            "args": ["/runscript=\"Grasshopper\""],
      },
```

I'm quite sure the args is correct, but I've never used an osx/windows condition? Would the above work?

---

These args should be added to the existing "osx" and "windows" sections, not create new ones just for args.  Yes it should work, though I think on Windows it should be "/runscript=_Grasshopper", and macOS "-runscript=_Grasshopper", with underscores so it is language agnostic.  This should also be applied to the VBGrasshopper template too.